tests/build_system_cflags_spaces/Makefile

```diff
@@ -1,4 +1,7 @@
 include ../Makefile.tests_common
+
+USEMODULE += test_utils_interactive_sync
```

Comment:
Is this really needed ? IIUC `test_utils_interactive_sync` is already pulled as default module from line 1.